plutus-core/plutus-ir/test/transform/inline/letNonPureMultiStrict

```diff
@@ -0,0 +1,21 @@
+{- 
+This test checks that impure bindings are not inlined by call site inlining.
+y is strict in this case.
+
+let y = trace "hello" 1
+    z = y
+    x = y -- so it looks small enough to inline
```

Comment:
since y is a strict variable this is in fact pure.  You want to make the binding for y non-strict in this case.